client/ayon_core/hosts/houdini/plugins/publish/validate_split_render_is_disabled.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+import hou
+from ayon_core.pipeline import PublishValidationError
+from ayon_core.pipeline.publish import RepairAction
+
+
+class DisableSplitExportAction(RepairAction):
+    label = "Disable Split Export"
+
+
+class ValidateSplitExportIsDisabled(pyblish.api.InstancePlugin):
+    """Validate the Instance has no current cooking errors."""
+
+    order = pyblish.api.ValidatorOrder
+    hosts = ["houdini"]
+    families = ["mantra_rop",
+                "redshift_rop",
+                "arnold_rop",
+                "vray_rop"]
+    label = "Validate Split Export Is Disabled"
+    actions = [DisableSplitExportAction]
+
+    def process(self, instance):
+
+        invalid = self.get_invalid(instance)
+        if invalid:
+            nodes = [n.path() for n in invalid]
+            raise PublishValidationError(
+                "See log for details. "
+                "Invalid nodes: {0}".format(nodes)
+            )
+
+
+    @classmethod
+    def get_invalid(cls, instance):
+
+        invalid = []
+        rop_node = hou.node(instance.data["instance_node"])
+
+        creator_attribute = instance.data["creator_attributes"]
+        farm_enabled = creator_attribute["farm"]
+        if farm_enabled:
+            cls.log.debug(
+                "Farm is enabled, skipping validation."
+            )
+            return
+
+
+        split_enabled = creator_attribute["split_render"]
+        if split_enabled:
+            invalid.append(rop_node)
+            cls.log.error(
+                "Split Export must be disabled in local render instances."
+            )
+
```

Comment:
I think we shouldn't use a validator for this. Instead - if local render is set we basically have full control over the "render" logic.We should just use a context manager around the `render_rop` call to ensure the settings are as we need to perform the actual render.

Reply:
Maybe we can do split render in somewhere after choosing the local render settings(as it is just for the farm submission). Like right before farm submission and after user's decision on whether they use local or farm rendering.

Reply:
Implemented in https://github.com/ynput/ayon-core/pull/328/commits/bca10c7c7d0d4d8457a2ba25730dd37ebd5bbff9